src/spicedb-common/services/developerservice.ts

```diff
@@ -226,16 +226,13 @@ class DeveloperServiceRequest {
       ),
     });
 
-    const developerRequest = JSON.stringify(
-      toJson(DeveloperRequestSchema, request),
-    );
+    const developerRequest = toJsonString(DeveloperRequestSchema, request);
```

Comment:
I also found this function while I was doing this work, and it seems like a better expression of what I'm trying to do.